dev-support/create-release/release-util.sh

```diff
@@ -423,17 +430,14 @@ function get_jira_name {
 
 # Update the CHANGES.md
 # DOES NOT DO COMMITS! Caller should do that.
+# requires yetus to have a defined home already.
```

Comment:
In order to make this use of YETUS_HOME work okay when not doing a Docker build, that is when invoking `do-release.sh` directly, it is necessary to collect the value of YETUS_HOME in `release_util.sh:get_release_info()`.  Perhaps you could add at line 232 something like:
```
  if [[ -z "$YETUS_HOME" && "$RUNNING_IN_DOCKER" != "1" ]]; then
    peer_dir="$(cd "$SELF/../../../ & echo `pwd`/yetus)"
    YETUS_HOME="$(read_config "YETUS_HOME" "$peer_dir")
    export YETUS_HOME
fi
```

Then in `do-release-docker.sh`, before the invocation of `get_release_info` in Line 116, add the line
```
RUNNING_IN_DOCKER=1
```

Reply:
let me go back and look. I was trying to make this equivalent to the JAVA_HOME we have already.

Reply:
take a look at the updated changeset please. we prompt now when running non-docker tag stage.